taxonomy-editor-frontend/src/pages/go-to-project/index.tsx

```diff
@@ -35,21 +35,18 @@ const GoToProject = ({ clearNavBarLinks }: Props) => {
     let newProjects: ProjectType[] = [];
 
     if (data) {
-      const backendProjects = data
-        .filter((projectsArrayItem) => projectsArrayItem[0])
-        .map((projectsArrayItem) => {
-          const { id, branch_name, taxonomy_name, description, errors_count } =
-            projectsArrayItem[0];
-
+      const backendProjects = data.map(
+        ({ id, branch_name, taxonomy_name, description, errors_count }) => {
           return {
-            id: Math.random().toString(), // Used as Material Table component key
+            id, // needed by MaterialTable as key
```

Comment:
Just to be sure, you're using the projectName attribute given by the API, as id for Material Table?

Reply:
@aadarsh-ram as every project is unique it seems ok, isn't it ? And it's more consistent.

Reply:
as far as I can see it is ok